Update index.md

maradeeym opened the pull request:
Added dRPC as third-party provider.

dRPC provides Rootstock RPC endpoints for Mainnet and Testnets including Websockets.

bguiz commented:
> Is this aimed for every rsksmart project or just for the devportal?

@alepc253 for now, just DevPortal. If it works well here, we can consider it for other repos/ teams.

If I had to guess, other teams are likely to want this concept, but are unlikely to want the same text inside their templates, and will want to customise this - so let's let that happen when we get there.

bguiz commented:
> Is this aimed for every rsksmart project or just for the devportal?

@alepc253 I did some quick research, and it looks like github does indeed support organisation-wide templates for issues, pull requests, and several other things. See https://help.github.com/en/github/building-a-strong-community/creating-a-default-community-health-file-for-your-organization

... so we have that option available to us should we choose to do so in the future. Of course other teams should be consulted first before we do this!

maradeeym committed:
Update index.md

Added dRPC as third-party provider.

dRPC provides Rootstock RPC endpoints for Mainnet and Testnets including Websockets.

owans commented:
Thank you for your contribution @maradeeym. We will review and merge.

owans reviewed (approved):
LGTM!
